variables.tf

```diff
@@ -27,6 +27,11 @@ variable "private_networks" {
   type        = list(string)
 }
 
+variable "terraform_state_bucket" {
+  description = "The name of the S3 bucket where Terraform state is stored."
+  type        = string
```

Comment:
```suggestion
  description = "The name of the S3 bucket where Terraform state is stored."
  nullable    = false
  type        = string
```

Reply:
Yup, good catch - I will have to add that elsewhere too.